config/config.go

```diff
@@ -61,7 +61,8 @@ type Config struct {
 	TxnLocalLatches  TxnLocalLatches `toml:"txn-local-latches" json:"txn-local-latches"`
 	// Set sys variable lower-case-table-names, ref: https://dev.mysql.com/doc/refman/5.7/en/identifier-case-sensitivity.html.
 	// TODO: We actually only support mode 2, which keeps the original case, but the comparison is case-insensitive.
-	LowerCaseTableNames int `toml:"lower-case-table-names" json:"lower-case-table-names"`
+	LowerCaseTableNames int   `toml:"lower-case-table-names" json:"lower-case-table-names"`
+	DbQPSMetricSwitch   int64 `toml:"db-qps-metric-switch" json:"db-qps-metric-switch"`
```

Comment:
1. Why not define it as a bool?
1. It is better to move it to https://github.com/pingcap/tidb/blob/41838ceb28d2f527cfaac539394c034f815663f0/config/config.go#L143

Reply:
ok